firefox-ios/firefox-ios-tests/Tests/XCUITests/PhotonActionSheetTests.swift

```diff
@@ -71,9 +71,9 @@ class PhotonActionSheetTests: BaseTestCase {
         mozWaitForElementToExist(app.otherElements["ActivityListView"].otherElements["example.com"])
         mozWaitForElementToExist(app.collectionViews.cells["Copy"], timeout: TIMEOUT)
 
-        var  fennecElement = app.collectionViews.scrollViews.cells.element(boundBy: 3)
+        var  fennecElement = app.collectionViews.scrollViews.cells.element(boundBy: 2)
         if iPad() {
-            fennecElement = app.collectionViews.scrollViews.cells.element(boundBy: 2)
+            fennecElement = app.collectionViews.scrollViews.cells.element(boundBy: 1)
```

Comment:
The indexes of the icon is updated.
![Simulator Screenshot - iPhone 15 Pro - 2023-12-12 at 14 48 51](https://github.com/mozilla-mobile/firefox-ios/assets/1740517/39e7f29e-6313-4c3e-a8d8-2a8e5a16aeb7)
![Simulator Screenshot - iPad Air (5th generation) - 2023-12-12 at 15 08 19](https://github.com/mozilla-mobile/firefox-ios/assets/1740517/2695bd08-d812-4d21-9837-1534e95b8595)